doc/source/GettingStarted/ConfiguringGafferForThirdPartyTools/index.md

```diff
@@ -0,0 +1,196 @@
+    
+    ```shell
+    user@desktop ~ $ source ~/.bash_profile
+    user@desktop ~ $ echo $ARNOLD_ROOT
+    !ARNOLD_PATH_LINUX!
+    ```
+
+4. The next time you start Gaffer, the Arnold nodes will be available from the node creation menu.
+    
+    <!-- TODO: ![Arnold node menu](images/arnoldNodes.png) -->
+
+
+### Arnold in OSX ###
+
+> Note :
+> For the purposes of this instruction, we will assume you have Arnold !ARNOLD_VERSION! installed to `!ARNOLD_PATH_OSX!`
+
+To create the `ARNOLD_ROOT` environment variable in OSX:
+
+1. Open `~/.bash_profile` with a text editor.
+
+2. Add the line `export ARNOLD_ROOT=!ARNOLD_PATH_OSX!` and save.
+
+3. In a terminal (Finder > Go > Utilities > Terminal), reload the user configuration file, and then verify the variable is set:
+    
+    ```shell
+    MacBook:~ user$ source ~/.bash_profile
+    MacBook:~ user$ echo $ARNOLD_ROOT
+    !ARNOLD_PATH_OSX!
+    ```
+
+4. The next time you start Gaffer, the Arnold nodes will be available from the node creation menu.
+    
+    <!-- TODO: ![Arnold node menu](images/arnoldNodes.png) -->
+
+
+## Configuring Gaffer for 3Delight ##
+
+For Gaffer to load 3Delight modules, a `DELIGHT` environment variable must point to the 3Delight installation directory. Before you begin, make sure that 3Delight is correctly installed and configured, and close any open instances of Gaffer.
+
+
+### 3Delight in Linux ###
+
+> Note :
+> For the purposes of this instruction, we will assume you have 3Delight !DELIGHT_VERSION! installed to `!DELIGHT_PATH_LINUX!`
+
+To create the `DELIGHT` environment variable in Linux:
+
+1. Open `~/.bash_profile` with a text editor.
+
+2. Add the line `export DELIGHT=!DELIGHT_PATH_LINUX!` and save.
+
+3. In a terminal, reload the user configuration file, and then verify the variable is set:
+    
+    ```shell
+    user@desktop ~ $ source ~/.bash_profile
+    user@desktop ~ $ echo $DELIGHT
+    !DELIGHT_PATH_LINUX!
+    
+    ```
+
+4. The next time you start Gaffer, the 3Delight nodes will be available from the node creation menu.
+    
+    <!-- TODO: ![Delight node menu](images/delightNodes.png) -->
+
+
+### 3Delight in OSX ###
+
+> Note :
+> For the purposes of this instruction, we will assume you have 3Delight !DELIGHT_VERSION! installed to `!DELIGHT_PATH_OSX!`
+
+To create the `DELIGHT` environment variable in OSX:
+
+1. Open `~/.bash_profile` with a text editor.
+
+2. Add the line `export DELIGHT=!DELIGHT_PATH_OSX!` and save.
+
+3. In a terminal (Finder > Go > Utilities > Terminal), reload the user configuration file, and then verify the variable is set:
+    
+    ```shell
+    MacBook:~ user$ source ~/.bash_profile
+    MacBook:~ user$ echo $DELIGHT
+    !DELIGHT_PATH_OSX!
+    ```
+
+4. The next time you start Gaffer, the 3Delight nodes will be available from the node creation menu.
+    
+    <!-- TODO: ![Delight node menu](images/delightNodes.png) -->
+
+
+## Configuring Gaffer for Tractor ##
```

Comment:
Andrew mentioned that only users at studios would need this information. Would it be prudent to move this into the "Configuring Gaffer at Your Studio" section?

Reply:
I would leave it in the third party tools section, because Tractor is a third party tool, and it's not necessarily limited to studios either (I presume that free RenderMan comes with it).